### PR TITLE
Users Data should be now saved in database

### PR DESCRIPTION
**Pull request by Asman97**

### CODE REVIEW

File: src/main/java/com/example/gradecalculator/entities/AllUsers.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package com.example.gradecalculator.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class AllUsers {
```
Comment (dEisenbeiss) [inline src/main/java/com/example/gradecalculator/entities/AllUsers.java:9]: needs to be renamed to User

File: src/main/java/com/example/gradecalculator/web/controller/FormController.java [hunk 2]
```diff
@@ -1,34 +1,37 @@
...
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseBody;
 
 @Controller
+@SpringBootApplication
 public class FormController {
```
Comment (dEisenbeiss) [inline src/main/java/com/example/gradecalculator/web/controller/FormController.java:14]: as noted in the other PR, this should be renamed to UserController

File: src/main/java/com/example/gradecalculator/web/controller/FormController.java [hunk 3]
```diff
@@ -1,34 +1,37 @@
...
+        AllUsers user = new AllUsers(name, email, message);
+        userRepository.save(user);
+        System.out.println("Name: " + name);
+        System.out.println("E-Mail: " + email);
+        System.out.println("Nachricht: " + message);
+        return "Daten wurden im Terminal angezeigt.";
+    }
 
-        return "Data has been saved in the backend.";
+    public static void main(String[] args) {
```
Comment (dEisenbeiss) [inline src/main/java/com/example/gradecalculator/web/controller/FormController.java:34]: why is there a main method in a controller?

File: src/main/java/com/example/gradecalculator/web/controller/FormController.java [hunk 4]
```diff
@@ -1,34 +1,37 @@
...
-import org.springframework.boot.autoconfigure.security.SecurityProperties;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseBody;
 
 @Controller
+@SpringBootApplication
```
Comment (dEisenbeiss) [inline src/main/java/com/example/gradecalculator/web/controller/FormController.java:13]: a controller should never be the entry point of your application

File: src/main/java/com/example/gradecalculator/entities/AllUsers.java (hunk 1 above)
Comment (Asman97) [reply src/main/java/com/example/gradecalculator/entities/AllUsers.java:9]: in last two branches its User, "Users Data should be now saved in database " was done before these branches

File: src/main/java/com/example/gradecalculator/web/controller/FormController.java (hunk 3 above)
Comment (Asman97) [reply src/main/java/com/example/gradecalculator/web/controller/FormController.java:34]: fixed

File: src/main/java/com/example/gradecalculator/web/controller/FormController.java (hunk 4 above)
Comment (Asman97) [reply src/main/java/com/example/gradecalculator/web/controller/FormController.java:13]: yup, fixed

**Pull request closed without merging**